spec/models/initiative_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+# == Schema Information
+#
+# Table name: initiatives
+#
+#  id          :integer          not null, primary key
+#  description :string
+#  startdate   :date
+#  status      :integer
+#  title       :string           not null
+#  type        :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  owner_id    :integer          not null
+#  parent_id   :integer
+#
+# Indexes
+#
+#  index_initiatives_on_owner_id   (owner_id)
+#  index_initiatives_on_parent_id  (parent_id)
+#  index_initiatives_on_title      (title) UNIQUE
+#
+# Foreign Keys
+#
+#  owner_id   (owner_id => wyeworkers.id) ON DELETE => cascade
+#  parent_id  (parent_id => initiatives.id)
+#
+
+RSpec.describe Initiative, type: :model do
+  describe "validations" do
+    let(:initiative) { build(:initiative) }
+
+    context "without an associated manager" do
+      it "is invalid" do
+        initiative.owner = build(:developer)
+        initiative.helpers = [build(:developer)]
+
+        initiative.validate
+        expect(initiative.errors.full_messages).to include(/manager/)
+      end
+    end
+
+    context "with an associated manager" do
+      describe "as the owner" do
+        it "is valid" do
+          initiative.owner = build(:manager)
+          initiative.helpers = [build(:developer)]
+          expect(build(:initiative, owner: build(:manager), helpers: [])).to be_valid
```

Comment:
Rarimel

Reply:
Hello,

I hope this message finds you well.

I wanted to inform you that I have received your comment regarding [specific issue or task]. I have addressed it accordingly, and the necessary actions have been taken.

Please feel free to reach out if you have any further questions or need additional information.

Thank you for bringing this to my attention.

Best regards,
Natalia